src/editor-state-prototypes.js

```diff
@@ -111,6 +112,14 @@ const stateSchema = {
       type: 'object',
       additionalProperties: languageSchema,
     },
+    copiedCells: {
+      type: 'array',
+      items: cellSchema,
+    },
+    cutCells: {
+      type: 'array',
+      items: cellSchema,
+    },
```

Comment:
is it necessary to have both copiedCells and cutCells (rather than, for example, just a single cellClipboard)? it may well be necessary, and perhaps i havenot understood the code deeply enough to see why both are needed.